docs/qasm.rst

```diff
@@ -121,3 +121,26 @@ standard gates and other included file is sufficient. However, note that a
 Braket.
 
 .. autofunction:: mpqp.qasm.qasm_to_braket.qasm3_to_braket_Program
+
+From OpenQASM to Cirq
+---------------------
+
+The Cirq library allows the user to instantiate a Cirq ``Circuit`` from an
+OpenQASM 2.0 code. This feature enables users to seamlessly transition 
+their quantum circuits from OpenQASM to Cirq, leveraging Cirq's capabilities
+and features.
+
+the Cirq parser lacks native support for certain OpenQASM 2.0 operations such as
+``cu1``, ``crz``, ``cu3``, ``reset``, ``u0``, ``p``, ``cp``, ``u``, ``rzz``, 
+``rxx`` and custom ``gate``. 
+To address this limitation, we've temporarily incorporated these gates directly 
+into the parser and for the custom gate we replace directly by native gate in 
+the qasm str. This ensures seamless generation of circuits from QASM code 
+using the qasm2_to_cirq_Circuit function.
+
+:func:`qasm2_to_cirq_Circuit<mpqp.qasm.qasm_to_cirq.qasm2_to_cirq_Circuit>` to generate
```

Comment:
Why do we add this line? The autofunction will generate the doc of the function, and in the docstring of the function we will have the goal of the function, which is generating circuit from qasm code. So we will have redundancy ?

Reply:
it's just the name of the function with a link to this
![image](https://github.com/ColibrITD-SAS/mpqp/assets/164006610/ec7954a1-b115-4ef9-8755-d854316e63e8)